components/dashboard/src/teams/TeamUsage.tsx

```diff
@@ -34,6 +34,7 @@ function TeamUsage() {
     const [startDateOfBillMonth, setStartDateOfBillMonth] = useState(timestampStartOfCurrentMonth);
     const [endDateOfBillMonth, setEndDateOfBillMonth] = useState(Date.now());
     const [isLoading, setIsLoading] = useState<boolean>(true);
+    const [startedTimeOrder] = useState(0); // Descending
```

Comment:
Can we use a smaller type rather than `number` here? ie a type like `'ascending' | 'descending'`.